maestro/main.go

```diff
@@ -454,10 +454,6 @@ func main() {
 	/*             Network startup               */
 	/*********************************************/
 	neterr := networking.InitNetworkManager(config.Network, config.DDBConnConfig)
```

Comment:
Can this line be moved to line 551, after the definition of bringUpIfs()?  I prefer to keep the distance between assignment and checking of a return code to as minimal as possible.